frontend/app/components/wp-relations/wp-relations-hierarchy/wp-relations-hierarchy.template.html

```diff
@@ -0,0 +1,19 @@
+<div class="wp-relations-hierarchy-section">
+    <div class="attributes-group--header">
+        <div class="attributes-group--header-container">
+            <h3 class="attributes-group--header-text">
+                {{::this.$ctrl.I18n.t('js.relations_hierarchy.hierarchy_headline')}}
```

Comment:
Save the translated text in a `text` attribute of the controller and use it like `$ctrl.text.hierarchyHeadline`. This way more readable and makes the template more reusable.